__snapshots__/snackbar.md

```diff
@@ -156,6 +156,7 @@
     </div>
```

Comment:
what's that got to do with anything?

Reply:
Don't really know - linter did that methinks. Can revert if needed.